artemis-server/src/main/java/org/apache/activemq/artemis/core/management/impl/ActiveMQServerControlImpl.java

```diff
@@ -1983,6 +1983,12 @@ public void forceFailover() throws Exception {
    }
 
    @Override
+   public void forceShutdown() throws Exception {
+      forceFailover();
+      System.exit(0);
```

Comment:
I'm not sure this operation is safe. It could be useful to destroy a server. Especially when you have embedded systems through Karaf, wildfly and others... 

I -1 for this feature



If shutdown the Artemis server is preventing the shutdown than it's a bug, but I don't think that's a case. If you see a case where the VM won't shutdown after a server.stop() then it's an issue.